internal/ceb/config.go

```diff
@@ -247,6 +262,27 @@ var sigMap = map[string]os.Signal{
 	"WINCH":  unix.SIGWINCH,
 }
 
+func (ceb *CEB) writeFiles(log hclog.Logger, cfg *config, env *appconfig.UpdatedConfig) {
+	log.Debug("writing application files to disk", "count", len(env.Files))
+
+	for _, fc := range env.Files {
+		err := ioutil.WriteFile(fc.Path, fc.Data, 0644)
+		if err != nil {
+			log.Error("error writing application file", "error", err, "path", fc.Path)
+		} else {
+			log.Info("wrote application file to disk", "path", fc.Path)
+		}
+	}
+
+	if cfg.FileRewriteSignal != "" {
+		if sig, ok := sigMap[strings.ToUpper(cfg.FileRewriteSignal)]; ok {
+			ceb.childSigCh <- sig
```

Comment:
Would we still want to continue with this if we errorerd on line 270?

Reply:
Totes right, I've fixed that. Thanks!